Packs/DemistoRESTAPI/Integrations/CoreRESTAPI/README.md

```diff
@@ -8,7 +8,7 @@
 
     | **Parameter** | **Description** | **Required** |
     | --- | --- | --- |
-    | Core Server URL |  | True |
+    | Core Server URL | For Cortex XSOAR 6.x, use the server URL. For Cortex XSOAR 8 and Cortex XSIAM, click the **COPY API URL** button on the API Keys page in **Integrations** > **API Keys** | True |
```

Comment:
Need to update the yml to have the same description https://github.com/demisto/content/pull/31930/files#diff-b2bf91e750c47a327f4c4cf45f76c35fe9fe6a661f0d39c0c20fde8cde98e513R14